docs/run/rest-api-primitives.mdx

```diff
@@ -0,0 +1,490 @@
+---
+title: Primitives using REST API
+description: How to use the Sampler and Estimator primitives with Qiskit Runtime REST API.
+---
+
+# Primitives using REST API
+
+The steps in this topic describe how to run primitives workloads with REST API, explore the options you can use to configure them, and invoke them in any program of your choice.
+
+<Admonition type="note">
+    This documentation utilizes the `requests` module of python to demonstrate the Qiskit Runtime REST API. However, this workflow can be executed using any language/framework that supports working with REST APIs. The official documentation for the API Reference can be found [here](https://docs.quantum.ibm.com/api/runtime/tags/jobs).
+</Admonition>
+
+<span id="start-estimator"></span>
+## Estimator Primitive with REST API
+
+### 1. Initialize the account
+
+Because Qiskit Runtime `Estimator` is a managed service, you first need to initialize your account. You can then select the simulator or real system you want to use to calculate the expectation value. You can then select the IBM Quantum Compute resource of your choice to run your calculations on. 
```

Comment:
```suggestion
Because Qiskit Runtime `Estimator` is a managed service, you first need to initialize your account. You can then select the device you want to use to calculate the expectation value. You can then select the IBM Quantum Compute resource of your choice to run your calculations on. 
```
we don't provide access to cloud simulators anymore (at least for the majority of users). Also we are supposed to be moving away from the term "system" unless we are specifically referring to System Two